kuma/core/views.py

```diff
@@ -56,3 +58,11 @@ def rate_limited(request, exception):
     response = render(request, '429.html', status=429)
     response['Retry-After'] = '60'
     return response
+
+
+@shared_cache_control(s_maxage=60 * 60 * 24 * 30)
+def humans_txt(request):
+    """We no longer maintain an actual /humans.txt endpoint but to avoid the
+    sad 404 we instead now just encourage people to go and use the GitHub
+    UI to see the contributors."""
+    return HttpResponse("See https://github.com/mdn/kuma/graphs/contributors\n")
```

Comment:
That's fine, but then I think we should make the URL a link for convenience.
```suggestion
    return HttpResponse('See <a href="https://github.com/mdn/kuma/graphs/contributors">https://github.com/mdn/kuma/graphs/contributors</a>\n')
```

Reply:
That'd require me to change the content-type to `text/html` which doesn't feel right considering it's `.txt`.